web/source/keyboards/kmwkeyboards.ts

```diff
@@ -965,6 +980,18 @@ namespace com.keyman.keyboards {
             }
 
             lList=x[i]['languages'];
+            if (!lList) {
```

Comment:
I'm not sure if this causes unit_tests to fail.
But this check is so lList.length isn't undefined in l.997